src/test/scala/nodes/learning/ZCAWhiteningSuite.scala

```diff
@@ -14,7 +14,7 @@ class ZCAWhiteningSuite extends FunSuite with LocalSparkContext with Logging {
     val ndim = 10
 
     val x = DenseMatrix.rand[Double](nrows, ndim, Gaussian(0.0, 1.0))
-    val whitener = new ZCAWhitenerEstimator(eps).fitSingle(x)
```

Comment:
So now `eps` defined above is not getting passed in -- should we just remove the eps definition above ?

Also we could add another test with eps=1e-3 and test whether the code is using it correctly ?